src/asciidoctorWebViewConverter.ts

```diff
@@ -69,9 +69,15 @@ function getCspForResource (webviewResourceProvider: WebviewResourceProvider, se
       return '<meta http-equiv="Content-Security-Policy" content="">'
 
     case AsciidocPreviewSecurityLevel.Strict:
-    default:
-      return `<meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src 'self' ${rule} https: data:; media-src 'self' ${rule} https: data:; script-src 'nonce-${nonce}' '${highlightjsInlineScriptHash}'; style-src 'self' ${rule} 'unsafe-inline' https: data:; font-src 'self' ${rule} https: data:;">`
-  }
+      // Initialize the cspRule variable with the default CSP rule
+      let cspRule = `default-src 'none'; img-src 'self' ${rule} https: data:; media-src 'self' ${rule} https: data:; script-src 'nonce-${nonce}' '${highlightjsInlineScriptHash}'; style-src 'self' ${rule} 'unsafe-inline' https: data:; font-src 'self' ${rule} https: data:;`;
```

Comment:
As a side note, we might want to use https://www.npmjs.com/package/content-security-policy-builder (or similar) to build CSP and avoid duplications/mistakes/typos.

If we don't want to use a library we could at least `join` to make it easier to read:

```js
const rules = [
  "default-src 'none'",
  `img-src 'self' ${rule} https: data:`,
  `media-src 'self' ${rule} https: data:`,
  `script-src 'nonce-${nonce}' '${highlightjsInlineScriptHash}'`,
  `style-src 'self' ${rule} 'unsafe-inline' https: data:`,
  `font-src 'self' ${rule} https: data:`
]
rules.join('; ')
```

Don't make this change in this pull request, that's just an idea to make it easier to maintain in the future.